BoomerangCommAudio/BoomerangCommAudio/Server.cpp

```diff
@@ -78,7 +189,47 @@ void RunServer(SOCKET& serverSock)
 	Display(temp);
 
 	PostMessage(g_hMainDlg, WM_CONNECT_SERVER, NULL, NULL);
+	*/
+
+}
+
+void AcceptFunc()
+{
+	int client_len;
+	struct	sockaddr_in client;
+	wchar_t temp[STR_SIZE];
+
+	while (true)
+	{
+		client_len = sizeof(client);
+		SOCKET acceptedSocket;
+		if ((acceptedSocket = accept(tcp_listen, (struct sockaddr *)&client, &client_len)) == INVALID_SOCKET)
+		{
+			wsprintf(temp, L"accept() failed with error %d", WSAGetLastError());
+			Display(temp);
+			break;
+		}
+		SocketInfo->Socket = acceptedSocket;
+		wsprintf(temp, L"Socket number %d connected", acceptedSocket);
+		Display(temp);
 
+		INFO_SONG infoSong;
+		infoSong.header = PH_INFO_SONG;
+		infoSong.SID = 1;
+		sprintf_s(infoSong.title, PACKET_STR_MAX, "%s", "Title of a song");
+		sprintf_s(infoSong.artist, PACKET_STR_MAX, "%s", "Artist of a song");
+		SocketInfo->DataBuf.buf = (char*)&infoSong;
+		SocketInfo->DataBuf.len = sizeof(INFO_SONG);
+		sendTCP(acceptedSocket, SocketInfo);
+
+		INFO_CLIENT infoClient;
+		infoClient.header = PH_INFO_CLIENT;
+		sprintf_s(infoClient.username, PACKET_STR_MAX, "%s", "luxes");
+		sprintf_s(infoClient.ip, IP_LENGTH, "%s", "192.168.0.22");
+		SocketInfo->DataBuf.buf = (char*)&infoClient;
+		SocketInfo->DataBuf.len = sizeof(INFO_CLIENT);
+		sendTCP(acceptedSocket, SocketInfo);
+	}
 }
 
 LRESULT CALLBACK ServerProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam, SOCKET& sock)
```

Comment:
Ignore this comment for now, we can figure that out later. 
Please get all the other corrections and ill merge.